### PR TITLE
Update GroupColumn.php

### PR DESCRIPTION
**Pull request by jimboobrien**
Added check for if exists for fieldID['before_row'] and fieldID['after_row']

### CODE REVIEW

File: Grid/Group/GroupColumn.php [hunk 1]
```diff
@@ -29,9 +29,12 @@ public function setColumnClassCmb2() {
 			//\Cmb2Grid\Cmb2\Utils::initializeFieldArg( $field->args['fields'][$fieldID], 'before_row' );
 			//\Cmb2Grid\Cmb2\Utils::initializeFieldArg( $field->args['fields'][$fieldID], 'after_row' );
 
-
-			@$field->args['fields'][ $fieldID ]['before_row'] .= "<div class=\"{$columnClass}\">";
-			@$field->args['fields'][ $fieldID ]['after_row']  .= '</div>';
+			if ( !empty( $fieldID['before_row'] ) ) {
```
Comment (jrfnl) [inline Grid/Group/GroupColumn.php]: Better to combine both checks into one `if` statement as having the closing html tag without the opening one or visa versa will create broken html.

File: Grid/Group/GroupColumn.php [hunk 2]
```diff
@@ -29,9 +29,12 @@ public function setColumnClassCmb2() {
...
+				@$field->args['fields'][ $fieldID ]['before_row'] .= "<div class=\"{$columnClass}\">";
```
Comment (jrfnl) [inline Grid/Group/GroupColumn.php]: As there is now an `empty()`  check around it, the `@` is no longer needed.

**Comment by jrfnl** [comment]
Thanks for this, this is something I've intended to change a number of times as well ;-)

**Comment by jimboobrien** [comment]
Sounds good, I made those changes.

**Pull request merged**